src/Linux/Host/Broker/Impl/Services/Utility.cs

```diff
@@ -131,5 +145,13 @@ public class Utility {
                 ContractResolver = new CamelCasePropertyNamesContractResolver()
             };
         }
+
+        private static IEnumerable<string> GetEnvironmentForForkAndExec(IDictionary<string,string> environment){
```

Comment:
This method can be replaced (where it's called, since it's only called once) with:
```cs
environment.Select(e => $"{e.Key}={e.Value}")
```